src/datanode/src/datanode.rs

```diff
@@ -148,20 +148,26 @@ impl From<&DatanodeOptions> for StorageEngineConfig {
 #[serde(default)]
 pub struct ProcedureConfig {
     /// Storage config for procedure manager.
-    pub store: ObjectStoreConfig,
+    pub store_path: String,
+    pub max_retry_times: usize,
+    pub retry_delay: u64,
```

Comment:
suggestion: Why not just use Duration?
```suggestion
    #[serde(with = "humantime_serde")]
    pub retry_delay: Duration,
```